tests/heartbeat.test.js

```diff
@@ -99,9 +102,26 @@ describe('heartbeat', () => {
         monitor.monitorHeartbeat();
       });
 
+      it('does not report down workers as stalled', () => {
+        supervisor.start();
+        Object.values(supervisor.workers).forEach((worker) => {
+          worker.state = 'down'; // eslint-disable-line no-param-reassign
+        });
+        const strId = Object.keys(supervisor.workers)[0];
+        monitor.heartbeatStartedAt = now; // timer won't be installed immediately
+        monitor.timestamps[strId] = [now[0] - 10, now[1]];
```

Comment:
what is 0 and 1 here? could we use more variables to make it clearer?

Reply:
implementation details of `hrtime` (probably seconds and nanoseconds or microseconds).  This reuses code from existing tests